files/scripts/sshd.fact

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+from re import match, IGNORECASE
+
+sshd_facts = [
+    { 'variable': 'AllowUsers',
+      'fact': 'allow_users',
+      'default': [] },
+    { 'variable': 'AllowGroups',
+      'fact': 'allow_groups',
+      'default': [] }
+]
+
+output = {'configured': 'true'}
+for item in sshd_facts:
+    output.update({item['fact']: item['default']})
+
+try:
+    with open("/etc/ssh/sshd_config") as fd:
+        for line in fd.readlines():
+            for item in sshd_facts:
+                match_fact = match(r"^" + item['variable'] + " ([ \w]+)", line, flags=IGNORECASE)
+                if match_fact:
+                    if isinstance(item['default'], list):
+                        output[item['fact']] = match_fact.groups()[0].split()
+                    elif isinstance(item['default'], str):
+                        output[item['fact']] = match_fact.groups()[0]
+                    elif isinstance(item['default'], int):
+                        output[item['fact']] = int(match_fact.groups()[0])
```

Comment:
Just saw this again and thought that there is probably a Python module which can do the parsing for us. And there is: [paramiko](https://packages.debian.org/jessie/python-paramiko). It has a SSHConfig class which should be preferred instead of home-brewed regex parsing :wink: